token-metadata/program/src/processor/edition/mint_new_edition_from_master_edition_via_token.rs

```diff
@@ -50,5 +50,6 @@ pub fn process_mint_new_edition_from_master_edition_via_token<'a>(
         },
         edition,
         ignore_owner_signer,
+        crate::state::TokenStandard::NonFungibleEdition,
```

Comment:
Yeah that makes sense. I must have moved this code around. At one point I was trying to avoid deserializing Metadata twice but it looks like that's not necessary anyway.